test/core/test/cli-test.test.ts

```diff
@@ -330,4 +330,13 @@ test('public parseCLI works correctly', () => {
       'color': true,
     },
   })
+
+  expect(parseCLI('vitest --sequence.shuffle.files --sequence.shuffle.tests=false')).toEqual({
```

Comment:
I would prefer if we had this in a separate test that tests all `shuffle` options. Please, don't use this specific test as it's only for the public `parseCLI` option, and it's already tested enough

Reply:
I've applied your suggestion, thanks for letting me know that.